api/src/paths/project/{projectId}/survey/{surveyId}/observations/create.ts

```diff
@@ -0,0 +1,152 @@
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { SYSTEM_ROLE } from '../../../../../../constants/roles';
+import { getDBConnection } from '../../../../../../database/db';
+import { HTTP400 } from '../../../../../../errors/CustomError';
+import { PostBlockObservationObject } from '../../../../../../models/block-observation-create';
+import { blockObservationIdResponseObject } from '../../../../../../openapi/schemas/observation';
+import { postBlockObservationSQL } from '../../../../../../queries/observation/observation-create-queries';
+import { getLogger } from '../../../../../../utils/logger';
+import { logRequest } from '../../../../../../utils/path-utils';
+
+const defaultLog = getLogger('paths/project/{projectId}/survey/{surveyId}/observations/create');
+
+export const POST: Operation = [
+  logRequest('paths/project/{projectId}/survey/{surveyId}/observations/create', 'POST'),
+  createBlockObservation()
+];
+
+POST.apiDoc = {
+  description: 'Create a new block observation.',
```

Comment:
this is not the correct description - this endpoint is generic for all observation creates not just block type